core/trino-main/src/main/java/io/trino/operator/GroupByIdBlock.java

```diff
@@ -47,91 +92,109 @@ public long getGroupCount()
 
     public long getGroupId(int position)
     {
-        return BIGINT.getLong(block, position);
+        checkReadablePosition(position);
+        return groups[position & mask];
+    }
+
+    @VisibleForTesting
+    boolean isRle()
+    {
+        return mask == MASK_RLE;
     }
 
     @Override
-    public Block getRegion(int positionOffset, int length)
```

Comment:
Is this method required?

```
 // This implementation cannot return region without copying
```
Can it have performance impact? You can actually make improved `GroupByIdBlock` support copyless `getRegion`.

Reply:
ping

Reply:
I don't think it is used. I removed it and let's wait for CI to finish

Reply:
> I don't think it is used. I removed it and let's wait for CI to finish

Please check the code too for possible paths where `getRegion` could be used